### DISCUSSION TITLE
零博客侵權內容

### DISCUSSION THREAD

**Discussion by kusakawazeusu** [agorahub/blog0#11] [General]
你好，

經查，零博客網站在未經授權前提下，擅自取用諸多方格子平台創作者發布之內容，其中包含付費限定內容被免費公開。

https://agora0.github.io/blog/vocus

皆未經方格子與創作者同意授權，擅自使用，此舉已經嚴重影響創作者與方格子平台的權利與聲譽，故特以此函通知。

請貴網立即撤除前述所有方格子平台相關內容及網址後回覆告知，以正視聽，並避免繼續侵害方格子與平台創作者之權益。

如有取用免費內容需求，請洽詢：editor@vocus.cc

以合法、符合規範、尊重的態度取用被允許授權內容。

請貴網妥善處理，以免訟累，特此告知。

方格子團隊 

2022-10-07

**Comment by theagora** [comment]
@agorahub/pen

**Comment by hokoi** [comment]
@agorahub 非商組織亦應跟進商業機構出具之意見。
惟應以理念相近爲前提，公開、透明拓展網絡。

**Reply by andyof** [reply]
商业利益诉求貌似贯穿题主行文，看不出有契合的理念诉求空间。

**Reply by hokoi** [reply]
> 商业利益诉求貌似贯穿题主行文，看不出有契合的理念诉求空间。

理解。即使應邀就“免費內容需求”“洽詢”所述商業機構，亦存在喪失獨立性等風險。

**Reply by andyof** [reply]
该网平台用户貌似可以自由转换“免费/付费”。技术上与题主所申诉商业利益存在无法预期的冲突的风险，除非获得某种豁免才能维持独立性。

**Reply by hokoi** [reply]
即是無以跟進。實質層面，雙方擱置跟進即是豁免。

**Reply by andyof** [reply]
现下考量，题主帖子是否代表对方持续跟进并可能对 @agorahub 造成实质影响。

**Reply by hokoi** [reply]
No news is good news.

**Comment by theagora** [comment]
### Follow-up Todos:
- [x] Step 1: Pause updates for 'sucoⓋ' - 7 Oct, 2022
- [x] Step 2: Collect ideas from community - "No news is good news."
- [x] Step 3: Resume updates for 'sucoⓋ' - 28 Oct, 2022
- [ ] Step 4: Collect feedback and opinions......
